docs/operate/mobility/move-arm/frame-how-to.md

```diff
@@ -0,0 +1,96 @@
+   You can think of it like this: The z-axis of the arm's origin frame is parallel to the z-axis of the world frame, and the origin of the arm's origin frame is in the same place as the origin of the world frame.
+
+   If you want to define the world frame as the corner of the table, leave the default values for now, and continue to the next step so that you know which way `x` and `y` point and can edit the configuration accordingly.
+
+1. Click **Save**.
+
+   By configuring your arm's frame like this, you have now defined the world frame as the same as the arm's base frame.
+
+### Figure out which way the arm's axes point
+
+The arm's base frame is now the world frame, but you do not know which way the x, y, and z axes of the arm frame point, unless you happen to be familiar with the arm's [kinematics file](/operate/reference/kinematic-chain-config/).
+
+1. On your arm's configuration card, click the **TEST** section (or use the **CONTROL** tab).
+
+1. Move the arm in each direction and note which way the arm moves.
+   For example, under the **MoveToPosition** section, move the arm in the positive Z direction.
+   If it moves upwards, then you know that the z-axis of the arm frame points upwards.
+1. If the directions do not match your intended world frame, you can edit the arm's `orientation` field to rotate the arm frame to match your intended world frame.
+   For example, if the arm's x-axis points to the right, but you want it to point to the left, you can set the `orientation` field to `"x"=0, "y"=0, "z"=1, "th"=180`.
+
+1. For your reference, mark the X, Y, and Z axes of the world frame on your workspace with tape or a marker.
+
+### Add a frame offset
+
+If you want to define the world frame as the corner of the table, you can now edit the arm's frame to be offset from the world frame by the distance from the table corner to the arm's base:
+
+1. Measure the X and Y distance from your designated table corner to the center of the arm's base.
+1. Edit the arm's frame configuration to set the `translation` field accordingly.
+   For example, if the arm's base is 100mm in the X direction and 200mm in the Y direction from the table corner, set the `translation` field to `"x"=100, "y"=200, "z"=0`.
+   If you chose not to align the arm's axes with the world axes, be sure to use the X, Y, and Z directions of the _world_ frame to set the `translation` field.
+1. Click **Save**.
+
+   The arm's frame is now offset from the world frame by the distance from the table corner to the arm's base.
+
+## Other ways to define the world frame
+
+If none of your components are fixed in place, it might make sense to define the world frame as the frame of the first component you add to your robot, for example, the mobile base.
```

Comment:
I think this should probably be earlier on this page when we talk about determining the world frame. Otherewise I have to get through a lot of info that may not suit me and I don't know that I need to get to this header. Maybe the header should be something about Mobile machines/robots ?

Reply:
Okay I'll change the header. However this is a less-well-supported case so I don't know that we want to highlight; see latest wording suggestion for why ^